lib/iex/lib/iex.ex

```diff
@@ -255,6 +255,28 @@ defmodule IEx do
   Number of expressions and their results to keep in the history.
   The value is an integer. When it is negative, the history is unlimited.
 
+  ## Documentation Helpers
+
+  Documentation Helpers allows iex to have a configurable set of modules
+  for looking up both Elixir and Erlang documentation. `:doc_helpers` is
+  a keyword list with these values.
+
+    * `:helpers` - A list of modules that implement the documentation
+                   functions. The default is `[IEx.H_Elixir, IEx.H_Erlang_Stub]`.
+                   Removing `IEx.H_Elixir` is not recommended.
+    * `:find`    - Can have two values `:first` or `:all` (:first is default)
+                   Determines whether all helpers are called or just
+                   the first helper that returns `:found`
```

Comment:
this was mentioned before i think, but it will not hurt to remind it again,
new lines should be idented with an extra 2 space, not to whatever spaces the previous line determines. if this works in the docs is sheer coincidence.

so the first block should read:
```elixir
    * `:helpers` - A list of modules that implement the documentation
      functions. The default is `[IEx.H_Elixir, IEx.H_Erlang_Stub]`.
      Removing `IEx.H_Elixir` is not recommended.
```